### ISSUE TITLE
Please keep up with upstream 😊

### ISSUE THREAD

**Issue by benhalpern** [LetsBuildGG/dev.to#1]
Hey folks, I just wanted to stop by to say that there is a lot of great work going into generalization and updates to the core codebase of dev.to (soon to be officially called forem), and the longterm health of this project will rely on keeping up to date.

If there's anything we can do to help you run on the main code instead of a fork, please let us know! Most customization can be done through `/internal/config` and we're constantly shipping new stuff like this.

Thanks!

**Comment by benhalpern** [comment]
Looks like you got up to date! Yay!!

**Comment by cpluta** [comment]
We really appreciate the work you and your team have been putting into making the site more and more configurable with each change. Also the new UI looks **_awesome_**!!!

The first time we tried it was a little painful not being familiar with github. We come from a TFS/AzureDevOps background so merging is a little different.

We finally found the sync_fork.sh and it helped a ton!

The thing I would say that would be helpful for others is knowing about particular upgrades for running locally like Ruby and Elastic Search. We've been checking the diff in the GemFile to identify those local dependencies.

However, we are still impressed by the platform every day. 😃
